.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## PR Title :boom:
+
+Please title this PR with a summary of the change, along with the JIRA card number.
+
+Suggested formats: 
+
+1. Fixes/Refs #RHIROS-XXX - Title
+2. RHIROS-XXX Title 
+
+Feel free to remove this section from PR description once done.
+
+## Why do we need this change? :thought_balloon:
+
+Please include the __context of this change__ here.
+
+## Documentation update? :memo:
+
+- [ ] Yes
+- [ ] No
+
+## Security Checklist :lock:
+
+Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)
```

Comment:
@saltgen just curious if we need to go through it for every PR? Can you share your experience around how you have been following this practice, it will be helpful for me to know 🙌🏻

Reply:
This was initially suggested by @kgaikwad, [here](https://github.com/RedHatInsights/ros-backend/pull/272#issuecomment-1307030465)
I vote for having this link here as well, since it is related to security.
One could skim through the `RedHatInsights/secure-coding-checklist`, preferably on raising every PR, and also have a look at Repo > Security tab > Code Scanning results (CodeQL) from time to time.

Reply:
Thanks for sharing details @saltgen sounds good 💯